tests/integration/telemetry/stats/prometheus/stats_filter_test.go

```diff
@@ -150,8 +150,6 @@ func setupConfig(cfg *istio.Config) {
 	}
 	// disable telemetry and mixer filter
 	cfg.Values["global.disablePolicyChecks"] = "true"
-	cfg.Values["global.MixerCheckServer"] = ""
-	cfg.Values["global.MixerReportServer"] = ""
```

Comment:
@richardwxn right now we want disableHttpReport=true too, unless telemetry.mixer.enable=false is taking care of it.

Reply:
these flags do nothing today, they aren't valid. It's already taken care of by the other settings